packages/testsweets/lib/src/ui/shared/busy_indecator.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:testsweets/src/ui/shared/shared_styles.dart';
+
+import 'app_colors.dart';
+
+class BusyIndecator extends StatelessWidget {
```

Comment:
This is spelled wrong. It should be `BusyIndicator`

Reply:
😁